copybookreader-plugins/src/main/java/co/cask/hydrator/plugin/batch/CopybookConstants.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.hydrator.plugin.batch;
+
+import com.google.common.collect.Lists;
+
+import java.util.List;
+
+/**
+ *Copybook Contants.
+ */
+public class CopybookConstants {
+
+  public static final int DEFAULT_FILE_STRUCTURE = net.sf.JRecord.Common.Constants.IO_FIXED_LENGTH;
+
+  //For this initial implementation, only fixed-length binary files will be accepted.
+  //Maininting a list to accomodate more file types in the future.
+  public static final List<Integer> SUPPORTED_FILE_STRUCTURES = Lists.newArrayList(
```

Comment:
we should remove this until it is necessary. It also should not be a List but an ImmutableSet.